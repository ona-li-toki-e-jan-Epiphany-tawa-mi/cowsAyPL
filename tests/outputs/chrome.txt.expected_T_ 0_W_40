/¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯\
| The only thing separ |
| ating the gonks from |
|  the chooms is how c |
| hromed'ya are        |
\______________________/
                    \                  
                     \                 
                       ^__^            
                       (oo)\_______    
                       (__)\       )\/\
                         0 ||----w |   
                           ||     ||   
