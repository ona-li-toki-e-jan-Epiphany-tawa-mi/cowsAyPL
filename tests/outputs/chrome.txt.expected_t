/¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯\
| + t The only thing separating the gonks  |
| from the chooms is how chromed'ya are    |
\__________________________________________/
                                        \                  
                                         \                 
                                           ^__^            
                                           (oo)\_______    
                                           (__)\       )\/\
                                               ||----w |   
                                               ||     ||   
