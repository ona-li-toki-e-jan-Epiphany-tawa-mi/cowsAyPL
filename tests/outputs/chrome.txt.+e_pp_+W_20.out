/¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯\
| The only thing separ |
| ating the gonks from |
|  the chooms is how c |
| hromed'ya are        |
\______________________/
        \
         \
           ^__^
           (pp)\_______
           (__)\       )\/\
               ||----w |
               ||     ||
